builtin/providers/aws/resource_aws_lb_cookie_stickiness_policy.go

```diff
@@ -53,11 +53,14 @@ func resourceAwsLBCookieStickinessPolicyCreate(d *schema.ResourceData, meta inte
 
 	// Provision the LBStickinessPolicy
 	lbspOpts := &elb.CreateLBCookieStickinessPolicyInput{
-		CookieExpirationPeriod: aws.Int64(int64(d.Get("cookie_expiration_period").(int))),
 		LoadBalancerName:       aws.String(d.Get("load_balancer").(string)),
 		PolicyName:             aws.String(d.Get("name").(string)),
 	}
 
+	if v := d.Get("cookie_expiration_period").(int); v > 0 {
```

Comment:
You can use `if v, ok := d.GetOk("cookie_expiration_period"); ok {` here, which will have pretty much the same effect, but will be slightly more readable.